app/design-system/design-system-hedvig/src/main/kotlin/com/hedvig/android/design/system/hedvig/HedvigBottomSheet.kt

```diff
@@ -101,21 +118,69 @@ private fun <T> InternalHedvigBottomSheet(
     containerColor = bottomSheetColors.bottomSheetBackgroundColor,
     contentColor = bottomSheetColors.contentColor,
     contentPadding = contentPadding,
-    dragHandle = {
-      DragHandle(
-        modifier = Modifier
-          .fillMaxWidth()
-          .wrapContentWidth(Alignment.CenterHorizontally)
-          .padding(top = 8.dp, bottom = 20.dp),
-      )
-    },
+    dragHandle = dragHandle
+      ?: {
+        DragHandle(
+          modifier = Modifier
+            .fillMaxWidth()
+            .wrapContentWidth(Alignment.CenterHorizontally)
+            .padding(top = 8.dp, bottom = 20.dp),
+        )
+      },
   ) {
     Column(modifier = Modifier.verticalScroll(rememberScrollState())) {
       content()
     }
   }
 }
 
+@Composable
+fun CrossSellDrugHandle(
+  contentPadding: PaddingValues,
+  modifier: Modifier = Modifier,
+  text: String? = stringResource(R.string.CROSS_SELL_BANNER_TEXT),
+) {
+  val direction = LocalLayoutDirection.current
+  Box(
+    modifier
+      .fillMaxWidth()
+      .height(40.dp)
+      .layout { measurable, constraints ->
+        val paddingStart = contentPadding.calculateStartPadding(direction).roundToPx()
+        val paddingEnd = contentPadding.calculateEndPadding(direction).roundToPx()
+        val adjustedConstraints = constraints.copy(
+          maxWidth = constraints.maxWidth + paddingEnd + paddingStart,
+          minWidth = constraints.minWidth + paddingEnd + paddingStart,
+        )
+        val placeable = measurable.measure(adjustedConstraints)
+        layout(placeable.width, placeable.height) {
+          placeable.place(0, 0)
+        }
+      }
+      .background(color = HedvigTheme.colorScheme.signalGreenFill),
+    contentAlignment = Alignment.Center,
+  ) {
+    Row(
+      verticalAlignment = Alignment.CenterVertically,
+    ) {
+      Icon(
+        HedvigIcons.Campaign,
+        contentDescription = EmptyContentDescription,
+        tint = HedvigTheme.colorScheme.signalGreenElement,
+        modifier = Modifier.size(20.dp),
+      )
+      Spacer(Modifier.width(8.dp))
+      if (text != null) {
```

Comment:
```suggestion
      if (text != null) {
        Spacer(Modifier.width(8.dp))
```
So that if the text is null the icon is not shifted to the left by 4.dp